models/gday/tests/testthat.R

```diff
@@ -0,0 +1,13 @@
+#-------------------------------------------------------------------------------
+# Copyright (c) 2012 University of Illinois, NCSA.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the
+# University of Illinois/NCSA Open Source License
+# which accompanies this distribution, and is available at
+# http://opensource.ncsa.illinois.edu/license.html
+#-------------------------------------------------------------------------------
+library(testthat)
+library(PEcAn.utils)
+
+logger.setQuitOnSevere(FALSE)
+test_check("PEcAn.MODEL")
```

Comment:
This line will fail. You need to change PEcAn.MODEL to PEcAn.GDAY. I'd noted the same bug in the pull request from @tonygardella , which you copied